datastore/statedata.go

```diff
@@ -153,6 +155,10 @@ var (
 		MenderStateUpdateControlPause:               "mender-update-control-pause",
 		MenderStateFetchUpdateControl:               "mender-update-control-refresh-maps",
 		MenderStateFetchRetryUpdateControl:          "mender-update-control-retry-refresh-maps",
+
+		// No longer used. If encountered in the database upon startup,
+		// they will result in the failure path, like most states do.
+		MenderStateReportStatusError: "status-report-error",
```

Comment:
If found it will go to rollback (when supported), which is not what we want. That was already the case before, but as we are deprecating the state maybe we should have a new `case` in init for deprecated states.

Reply:
Right, that makes sense. I'll just send it to the Idle state then, the same as what would have happened during normal execution.